x-pack/legacy/plugins/fleet/public/pages/agent_list/index.scss

```diff
@@ -0,0 +1,6 @@
+.fleet__agentList__table .euiTableFooterCell {
+  .euiTableCellContent,
+  .euiTableCellContent__text {
+    overflow: visible;
```

Comment:
Why this is needed?

Reply:
the cells of the table footer is always truncated by EUI (aka `overflow: hidden;`), this lets the select message and toggle link be visible since they overflow the cell width